### PR TITLE
[TS] LPS-185987 | DDL entry deletion is not published to Live via the Staging option of the DDL portlet

### PR DESCRIPTION
**Pull request by bakayattila**
Hi Team,
Could you review this?

https://liferay.atlassian.net/browse/LPS-185987
Best,
Attila

-----------------------------------

I created a new fix. I returned to my original approach to fix the issue in the DDLDisplayPortletDataHandler. The layout could be displayed differently by modifying publish_portlet_publish_or_copy.jsp similarly to #359, but the actual delete event was not exported by only this modification. As I had to modify the DDLDisplayPortletDataHandler anyway I returned to my original approach. The fix behaves in a similar manner to the BlogsPortlet and redirects some actions to the related adminPortlets.

### CODE REVIEW

File: modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/exportimport/data/handler/DDLDisplayPortletDataHandler.java [hunk 1]
```diff
@@ -51,12 +57,37 @@ public String getServiceName() {
 		return DDLConstants.SERVICE_NAME;
 	}
 
+	@Override
+	public boolean isDisplayPortlet() {
+		return false;
```
Comment (JoyceWang08211) [inline modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/exportimport/data/handler/DDLDisplayPortletDataHandler.java:62]: Unique Integration [failure](https://testray.liferay.com/reports/production/logs/2023-07/test-1-25/test-portal-acceptance-pullrequest(master)/2179/modules-integration-mysql57-jdk8/0/1/jenkins-console.txt.gz?authuser=0):
` [exec] com.liferay.dynamic.data.lists.internal.exportimport.data.handler.test.DDLDisplayPortletDataHandlerTest > testIsDisplayPortlet FAILED
     [exec]     java.lang.AssertionError: expected:<true> but was:<false>`

Hi @bakayattila and @vendeltoreki,
Could you please take a look? Thx

**Comment by bakayattila** [comment]
c:test:sf

**Comment by bakayattila** [comment]
ci:test:relevant

**Comment by JoyceWang08211** [comment]
ci:test:relevant

**Comment by JoyceWang08211** [comment]
No regression was found.

**Comment by vendeltoreki** [comment]
ci:forward:force

**Pull request closed without merging**